src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```diff
@@ -264,6 +264,8 @@ def show_feature(cmd,
         raise CLIErrors.ResourceNotFoundError("Feature '{}' with label '{}' does not exist.".format(feature, label))
     except HttpResponseError as exception:
         raise CLIErrors.AzureResponseError(str(exception))
+    except Exception as exception:
+        raise CLIError(str(exception))
```

Comment:
Why do we catch base Exception and rethrow as CLIError ?

Reply:
Discussed offline, this is to ensure that the full stack trace of the error is not logged and only the direct cause is shown.